torch/_torch_docs.py

```diff
@@ -1022,7 +1022,7 @@ def parse_kwargs(desc):
 
 add_docstr(torch.cumsum,
            r"""
-cumsum(input, dim, out=None) -> Tensor
+cumsum(input, dim, out=None, dtype=None) -> Tensor
```

Comment:
Why did this one change?

Reply:
Because cumsum actually has dtype argument, and docs don't show it, it's a bug in docs.